avm/res/network/application-gateway/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -467,15 +463,7 @@ module testDeployment '../../../main.bicep' = [
           }
         }
       ]
-      webApplicationFirewallConfiguration: {
-        enabled: true
-        fileUploadLimitInMb: 100
-        firewallMode: 'Prevention'
-        maxRequestBodySizeInKb: 128
-        requestBodyCheck: true
-        ruleSetType: 'OWASP'
-        ruleSetVersion: '3.0'
-      }
+      firewallPolicyId: nestedDependencies.outputs.fwPolicyId
```

Comment:
While a bit out of scope for the purpose of this PR, it might be a good call to use the oppoertunity and also 'fix' the naming of this parameter and call it `firewallPolicyResourceId`

Reply:
Great call - will do it now.

Reply:
Latest commit now has the update. :)